OrbitVulkanLayer/PhysicalDeviceManager.h

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) 2020 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef ORBIT_VULKAN_LAYER_PHYSICAL_DEVICE_MANAGER_H_
+#define ORBIT_VULKAN_LAYER_PHYSICAL_DEVICE_MANAGER_H_
+
+#include "OrbitBase/Logging.h"
+#include "absl/container/flat_hash_map.h"
+#include "absl/synchronization/mutex.h"
+#include "vulkan/vulkan.h"
+
+namespace orbit_vulkan_layer {
+
+/*
+ * This class maintains a mapping from logical to physical devices (via `vkCreateDevice` and
+ * `vkDestroyDevice`).
+ * `TrackPhysicalDevice` establishes that mapping, while `UntrackLogicalDevice` can used to release
+ * that mapping. To retrieve a logical device's physical device, use `GetPhysicalDevice`.
+ *
+ * For each physical device, it also provides the `VkPhysicalDevice`, which can be queried using
+ * `GetPhysicalDeviceProperties`. This class is responsible to retrieve this information
+ * (using `vkGetPhysicalDeviceProperties`). These properties can be e.g. used for converting clock
```

Comment:
Isn't the sentence "This class is responsible to retrieve this information (using `vkGetPhysicalDeviceProperties`)" just a duplication of the previous one? (Btw, "responsible for retrieving")

Reply:
Or maybe I got it wrong. What does "this class" refer to here? `VkPhysicalDevice`?

Reply:
It was reffering to the DeviceManager class. I renamed it.